src/ucp/proto/proto_select.h

```diff
@@ -43,6 +45,11 @@
 #define UCP_PROTO_SELECT_PARAM_STR_MAX 128
 
 
+/* Pack operation attributes from uint32_t to a uint8_t */
+#define ucp_proto_select_op_attr_pack(_op_attr) \
+    (((_op_attr) & UCP_PROTO_SELECT_OP_ATTR_MASK) / \
+    UCP_PROTO_SELECT_OP_ATTR_BASE)
```

Comment:
pls check code style in clang format, seems like hav alignment issue

Reply:
I'll check why, thank you, done